### ISSUE TITLE
Cant get logged params into the web ui

### ISSUE THREAD

**Issue by paulgureghian** [mlflow/mlflow#652]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in MLflow)**:
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
- **MLflow installed from (source or binary)**: 
- **MLflow version (run ``mlflow --version``)**:
- **Python version**: 
- **npm version (if running the dev UI):
- **Exact command to reproduce**:

### Describe the problem
I am able to to open the web ui, but cant get the logged params to show there. 
I have mlflow installed and imported

### Source code / logs
~~~
with mlflow.start_run(nested =True):
### instantiate the RNN model object 
    regr = Sequential() 

### add the input and LSTM layers 
regr.add(LSTM(units =4, activation ='sigmoid', input_shape =(None, 1)))    

### add the output layer
regr.add(Dense(units =1))

### compile the RNN 
optimizer = 'adam'
loss = 'mean_squared_error'
metrics = ['accuracy'] 
regr.compile(optimizer, loss , metrics) 

### fit the model on the training set 
batch_size = 5
epochs = 1
regr.fit(X_train, y_train, batch_size, epochs) 

log_param("loss", loss) 
log_param("epochs", epochs)
log_param("optimizer", optimizer)
log_param("batch_size", batch_size)
log_param("metrics", metrics)
~~~
![screen shot 2018-10-18 at 5 50 20 pm](https://user-images.githubusercontent.com/8146561/47244209-7cb57280-d3a9-11e8-87ae-5490164863b7.png)

**Comment by DorIndivo** [comment]
The log_param method should be in the "with" block.
for example:
```
with mlflow.start_run():
    mlflow.log_param("a", 1)
    mlflow.log_metric("b", 2)
```

**Comment by paulgureghian** [comment]
@DorIndivo   Are you still here ?

**Comment by DorIndivo** [comment]
Yea

**Comment by paulgureghian** [comment]
Thanks. I have one more question.  just a minute

**Comment by paulgureghian** [comment]
![capture3](https://user-images.githubusercontent.com/8146561/47942062-dfc8fe00-dead-11e8-8d9b-effd5e382912.PNG)

**Comment by paulgureghian** [comment]
![capture4](https://user-images.githubusercontent.com/8146561/47942085-f0797400-dead-11e8-9407-fc8e4b69ca54.PNG)

**Comment by paulgureghian** [comment]
As you can see I used the  with mlflow.start_run()   for params and metrics for two separate runs. the params are in my web ui, but I cant get the log_metric() to show in the ui.

**Comment by paulgureghian** [comment]
I used the variable 'history' to store the output from model.compile, then a function to get the numeric values to pass to log_metric().  problem is , the 'history' variable is not getting anything stored in it. how to get log_metric()  to work ?

**Comment by DorIndivo** [comment]
Do you mean that the 'history' variable is None?

**Comment by paulgureghian** [comment]
yes. thats seems to be the case

**Comment by DorIndivo** [comment]
according to the documentation you should pass a float value for the log_metric function.
https://www.mlflow.org/docs/latest/python_api/mlflow.html#mlflow.log_metric

**Comment by paulgureghian** [comment]
yeah, i think i saw that. the value i see in the console is a float though.

**Comment by paulgureghian** [comment]
Epoch 1/1
2337/2337 [==============================] - 6s 3ms/step - loss: 0.1676 - mean_absolute_error: 0.3589

**Comment by paulgureghian** [comment]
my history variable is getting nothing though

**Comment by DorIndivo** [comment]
what is the type of 'history' variable ?

**Comment by DorIndivo** [comment]
I don't think this is the right place for this kind of personal problem.
I can help you privately.

**Comment by paulgureghian** [comment]
I can invite you to my slack channel. what is your email ? or do you have some other way ?

**Comment by apurva-koti** [comment]
Keep in mind that issues are not for extended back-and-forth conversation; keeping your messages short and succinct will help others following the issue. Thanks!
